setup.py

```diff
@@ -25,7 +25,7 @@ def get_version():
     url="https://github.com/executablebooks/sphinx-tabs",
     license="MIT",
     python_requires="~=3.7",
-    install_requires=["sphinx", "pygments", "docutils~=0.18.0"],
+    install_requires=["sphinx", "pygments", "docutils>=0.18.0"],
```

Comment:
```suggestion
    install_requires=["sphinx", "pygments", "docutils"],
```
I'd prefer to remove the pin entirely, so we don't have to bump it in future?

Sorry for the slow reply! Can get this merged if you think this makes sense

Reply:
Because this is a lower bound, we should not need to touch it in future. My original commit just ensures that we define a safe _minimum_ version that we know is the limit of our support.

So, I don't think we need your amendment, but feel free to challenge this if you have reason to want to e.g. widen the support to older docutils?

Reply:
@foster999 I know it's Christmas/holiday period, so I'm not expecting to hear from you any time soon. But, in case you are tinkering over the festive period, are you happy for me to merge this as is?

Reply:
Sorry for the slow reply! I've removed the lower bound just for consistency with how we treat sphinx and pygments. Happy for it to be merged now 😊